staff/francisco-fernandez/cv-styled/styles.css

```diff
@@ -0,0 +1,115 @@
+header{
+    padding:10px;
+    text-align:center;
+}
+img{
+    width: 150px;
+    height:150px;
+}
+
+footer{
+    text-align:center;
+    font-size:75%;
+}
+
+.leftcontainer{
+   background-color:rgb(115, 115, 141) ; 
+   text-align:center;
+   border-radius:10px;
+}
+
+.rightcontainer{
+   background-color: rgb(240, 235, 221) ; 
+   text-align:justify;
+   border-radius:10px;
+   padding-right:5px;
+}
+.grid-container {
+    display: grid;
+    grid-template-columns: repeat(12,1fr);
+    grid-template-rows: auto;
+    padding: 10px;
+    width: auto;
+  }
+
+  /* For mobile phones: */
+    [class*="item"] {
+        grid-column-start: 2;
```

Comment:
This is good, but you might want to consider start using the shorthand: 
grid column: start / end 
that way you can have less code

Reply:
I've changed it, yes it´s better, less code